misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
@dbkinder do we need a license header here?

Reply:
We should. Will add in v2.